docker/docker-blockchain-link-test.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -e
+
+xhost +
```

Comment:
does it have any visible UI? if not, this should not be needed

Reply:
You are right! Done!